src/js/components/ServiceForm.js

```diff
@@ -40,6 +40,8 @@ const FIELDS_TO_WATCH = {
   }
 };
 
+let networkTypeChanged = false;
```

Comment:
@MatApple consider that this fix is for a patch version and this code probably doesn't exist in the same form on our `master` branch.